### PR TITLE
Revise and update front end variable use

### PR DESCRIPTION
**Pull request by austinstarin**

### CODE REVIEW

**Comment by austinstarin** [comment]
Nudging @abea @bgantick

**Comment by abea** [comment]
Maybe choose a different example since I think elsewhere we discourage event handlers on `body`, right?

**Comment by austinstarin** [comment]
@bgantick @abea 72a2bd2

**Comment by austinstarin** [comment]
@abea @bgantick and da187f1

File: javascript.md [hunk 1]
```diff
@@ -686,52 +690,3 @@ self.$el.find('.my-thing').on('click', function() {
```
Comment (bgantick) [inline javascript.md]: This seems contrary to added example above now..

Comment (austinstarin) [inline javascript.md]: Leaving for now — it's in the "needs review" section and we can address as we continue to clean up.

**Pull request merged**